src/shared-components/typography/index.ts

```diff
@@ -43,6 +43,12 @@ const bodyStyle = (theme: ThemeType) => `
   ${baseBodyStyles(theme)}
 `;
 
+const bodyBoldStyle = (theme: ThemeType) => `
+  ${bodyStyle(theme)}
+  font-weight: ${theme.TYPOGRAPHY.fontWeight.bold};
```

Comment:
`font-weight` added for direct `css` usage if we need to use it instead of `Typography.Bold` (which is `<strong>`)